brands/amenity/restaurant.json

```
@@ -174,6 +174,17 @@
       "name:he": "בלאק"
     }
   },
+  "amenity/restaurant|Black Angus": {
+    "countryCodes": ["en"],
+    "tags": {
+      "amenity": "restaurant",
+      "brand": "Black Angus",
+      "brand:wikidata": "Q4920269",
+      "brand:wikipedia": "en:Black_Angus_Steakhouse",
```

these links need underscores replaced with spaces..
```suggestion
      "brand:wikipedia": "en:Black Angus Steakhouse",
```

---

ok